app/assets/stylesheets/application.scss

```diff
@@ -71,7 +71,7 @@ trix-toolbar .trix-button-row {
   flex-wrap: wrap;
 }
 
-.kaviar-sidebar {
+.lesson_material-sidebar {
```

Comment:
Where is this one actually used? There is no direct occurrence, so is it constructed dynamically somewhere?
If used, can you replace `lesson_material-sidebar` by `lesson-material-sidebar` to conform to CSS naming conventions?